### DISCUSSION TITLE
How to resize the logo?

### DISCUSSION THREAD

**Discussion by rgrajan** [uPagge/uBlogger#86] [Q&A] [answered]
Where should the logo can be resized dynamically? I tried playing around with .logo-svg at _ublogger.scss but unable to do that. Any help would be appreciated.  If i specify image width, then it will be constant for all devices. Currently it looks like this
![image](https://user-images.githubusercontent.com/7380733/115979195-8f006080-a552-11eb-8bf1-59d56e9b5152.png)

**Comment by uPagge** [comment]
Why change it dynamically, you just need to use one size that fits the hat and that's it

![изображение](https://user-images.githubusercontent.com/40397740/115983884-a835fb00-a5ac-11eb-9678-9c49948c1dff.png)

By the way, it seems to be not documented anywhere, but you can set your logo like this. This is the approach I use on my site.

Regarding your question directly, it is not clear what you want to get in the end.

**Comment by uPagge** [comment, marked as answer]
Most likely you do not need to dynamically resize, you need to resize once to 93x30 and that's it

**Reply by rgrajan** [reply]
Thanks @uPagge. My logo size was 5000x1338 so thought of changing dynamically based on device types. I have resized the image to 93x30 as you mentioned for now.
